tokio-console/src/lib.rs

```diff
@@ -12,7 +12,7 @@
 /// `tokio-console`:
 ///
 /// ```text
-#[doc = include_str!("../args.example")]
```

Comment:
Could we also delete `args.example` please.

Reply:
I think I already did:
<img width="938" alt="image" src="https://github.com/tokio-rs/console/assets/29879298/0d78b164-27c3-46c1-b89d-ca96b6f6d8ed">

Reply:
You're right! Sorry, I missed that it was renamed.

This is probably fine to go then. I'm going to give it a once over from my computer and then it can probably be merged.

Thanks for your contributions!